src/Core/Worker.php

```diff
@@ -51,26 +52,29 @@ class Worker
 	private static $last_update;
 	private static $state;
 	private static $daemon_mode = null;
+	/** @var Worker\Entity\Process */
```

Comment:
The added use statement enables using the shorthand
```suggestion
	/** @var Process */
```